lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/JavaBridge.java

```diff
@@ -18,6 +18,9 @@
  */
 package org.apache.reef.javabridge;
 
+/**
+ * TODO[JIRA REEF-383] Document/Refactor JavaBridge.
```

Comment:
I've seen TODOs formatted as `TODO[REEF-###]` as well. What is the standard format?